### PR TITLE
Basic setup to validate PR

### PR DESCRIPTION
**Pull request by willianantunes**

### CODE REVIEW

**Review by ricardochaves — APPROVED** [review]

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: ricardochaves/python-lint@v1.1.0
+        with:
+          python-root-list: "python_fluentd_testing tests"
+          use-black: true
+          use-mypy: true
+          use-isort: true
```
Comment (ricardochaves) [inline .github/workflows/lint.yml:14]: true is default

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          use-pycodestyle: false
+          use-pylint: false
+          use-flake8: false
+          extra-black-options: "--diff"
+          extra-pycodestyle-options: ""
+          extra-flake8-options: ""
+          extra-mypy-options: ""
```
Comment (ricardochaves) [inline .github/workflows/lint.yml]: you dont need "" value here

Comment (willianantunes) [reply .github/workflows/lint.yml]: @ricardochaves that's good to know. I copied from a project of ours. I'll change it.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (willianantunes) [reply .github/workflows/lint.yml:14]: Good to know it! I believe I'll leave them that way to be explicit about what I want. I would have to see the project to understand it or see previous builds.

**Pull request merged**